index.d.ts

```diff
@@ -175,7 +176,7 @@ interface EntityMp {
 		boneIndex: number, isRel: number, p9: boolean, highForce: boolean, p11: boolean, p12: boolean): void;
 	applyForceToCenterOfMass(forceType: number, x: number, y: number, z: number, p4: boolean, isRel: boolean,
 		highForce: boolean, p7: boolean): void;
-	attachTo(entity: Handle, boneIndex: number, xPos: number, yPos: number, zPos: number, xRot: number, yRot: number, 
```

Comment:
What about seperate commits? Though I know I am a bit nit-picking by saying so but it is clearer for everybody what has changed (as your PR only says that you'll add the ConsoleMp typing. Nvm.